deployment/kubernetes/yaml/mysql.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-mysql
+  labels:
+    name: ftgo-mysql
+spec:
+  ports:
+  - port: 3306
+    targetPort: 3306
+  clusterIP: None
+  selector:
+    role: ftgo-mysql
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ftgo-mysql
+spec:
+  selector:
+    matchLabels:
+      role: ftgo-mysql
+  serviceName: "ftgo-mysql"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        role: ftgo-mysql
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: ftgo-mysql
+          image: msapatterns/mysql:latest

```

Comment:
Please use the image from `eventuate-tram-examples-customers-and-orders/docker-compose-mysql-binlog.yml`

Reply:
updated